python/fapolicy_analyzer/tests/test_analyzer_selection_dialog.py

```diff
@@ -18,9 +23,42 @@ def test_adds_dialog_to_parent():
     assert widget.get_ref().get_transient_for() == parent
 
 
-def test_trust_database_admin_selection():
-    widget = AnalyzerSelectionDialog(Gtk.Window())
+def test_trust_database_admin_selection(widget):
     trustAdminBtn = widget.get_object("adminTrustDatabasesBtn")
     delayed_gui_action(trustAdminBtn.clicked)
     result = widget.get_selection()
     assert result == ANALYZER_SELECTION.TRUST_DATABASE_ADMIN
+
+
+def test_analyze_from_audit(widget):
+    widget.get_object("auditLogTxt").set_text("foo")
+    delayed_gui_action(widget.get_object("analyzeAuditBtn").clicked)
+    result = widget.get_selection()
+    assert result == ANALYZER_SELECTION.ANALYZE_FROM_AUDIT
+    assert widget.get_data() == "foo"
+
+
+@pytest.mark.parametrize(
+    "response_type", [Gtk.ResponseType.OK, Gtk.ResponseType.OK, Gtk.ResponseType.CANCEL]
+)
+def test_set_audit_file_from_dialog(widget, mocker, response_type):
```

Comment:
Is response_type used within this unit test?

Reply:
I removed it.